src/Operations.jl

```diff
@@ -961,21 +983,57 @@ function assert_can_add(ctx::Context, pkgs::Vector{PackageSpec})
     end
 end
 
+function tiered_resolve(ctx::Context, pkgs::Vector{PackageSpec})
+    try # do not modify existing subgraph
+        printpkgstyle(ctx, :Resolve_Tier, "All dependency versions.")
```

Comment:
Presumably this is debug output and won't shown to the user later? At least I don't think we should show it.

Reply:
Agreed. Its better without the user messages